### ISSUE TITLE
Add inline comments to the diff

### ISSUE THREAD

**Issue by pbu88** [pbu88/diffy#3]
Everything should be via Ajax. A user will click in a line and the table will expand to show a text area and a text input to include the name. After saving, the comment should show up to everyone. The comment should be markdown syntax

- [ ] Investigate how a library like AngularJS could help us in the frontend work
- [ ] Create html style for the comments, the conversation should be stacked
- [ ] Create all the frontend logic for the comments
- [ ] Find out a library and create the code to parse Markdown (frontend better, but not sure)
- [ ] Create the views for the backend
- [ ] Create the code that stores the comment in the database
- [ ] The comments will be stored in the mongo doc under `inlineComments` field
  - Basically it should be something like:
         ```
         inlineComments: {
             line: <line number>
             filename: <filename>  // btw, I'm thinking how to add ID's to the files, it would be so much better
             name: <the name of the one that commented>
             text: <the text of the comment>
             created: <the date of the comment>
         }

         ```
- [ ] Comments should appear in real time for multiple users viewing the diff at the same time (no need to refresh)

**Comment by pbu88** [comment]
Closing for now since this is not a priority currently.
